distributionviewer/core/static/js/app/reducers/metric-metadata-reducer.js

```diff
@@ -4,15 +4,31 @@ import * as types from '../actions/action-types';
 const initialState = {
   isFetching: false,
   metadata: [],
+  hoverStrings: [],
   status: 200
 };
 
+const getHoverStrings = (metadata) => {
+  let strings = [];
```

Comment:
Would it make sense for `strings` to be an object? Using an array works, but it might use more memory than we need if there's a big gap between `metricId`s. For example:

```javascript
$ node
> const sites = [];
> sites[1] = "Twitter";
> sites[48939] = "Amazon";
> sites.length
48940
```